extensions/extensions-security/extensions-policy-decision-point-service/src/main/resources/quarkus-app.xml

```diff
@@ -0,0 +1,29 @@
+<!--
+  #%L
+  AIOps Docker Baseline::Versioning::Service
+  %%
+  Copyright (C) 2021 Booz Allen
+  %%
+  This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+  #L%
+  -->
+<assembly
+	xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
+	<id>app</id>
+
+	<includeBaseDirectory>false</includeBaseDirectory>
+
+	<formats>
+		<format>zip</format>
+	</formats>
+
+	<fileSets>
+		<fileSet>
+			<directory>${project.build.directory}/quarkus-app</directory>
+			<outputDirectory>quarkus-app/</outputDirectory>
+		</fileSet>
+	</fileSets>
+
+</assembly>
```

Comment:
A: Think this file is using tabs instead of spaces.

Reply:
Ah yeah I'll update that real quick!